packaging/docker/build-images.sh

```diff
@@ -39,43 +38,34 @@ function create_fake_website_directory() {
         "unstripped_artifactory")
             logg "DOWNLOADING BINARIES TAR FILE"
             curl -Ls "${artifactory_base_url}/${fdb_version}/release/api/foundationdb-binaries-${fdb_version}-linux.tar.gz" | tar -xzf -
+            # Add the x86_64 to all binaries
+            for file in "${fdb_binaries[@]}"; do
+                cp -pr "${file}" "${file}.x86_64"
```

Comment:
I think this should be an `mv` and not `cp` unless there is a reason to have files with and without the `.x86_64` extension.